### PR TITLE
Ag 6686/theme tweaks

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-9474, https://ag-grid.atlassian.net/browse/AG-9473, https://ag-grid.atlassian.net/browse/AG-9457, https://ag-grid.atlassian.net/browse/AG-6686

### CODE REVIEW

**Review by alexanderby — APPROVED** [review]
Perfectum!

File: packages/ag-charts-community/src/chart/themes/chartTheme.ts [hunk 1]
```diff
@@ -291,13 +300,17 @@ export class ChartTheme {
         number: {
             ...ChartTheme.getAxisDefaults(),
             line: {
+                ...ChartTheme.getAxisDefaults().line,
+                // enabled: false, TODO: fix breaking tests when this is set, use width: 0 for now
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/themes/chartTheme.ts:304]: ?

File: packages/ag-charts-community/src/chart/themes/chartTheme.ts [hunk 2]
```diff
@@ -291,13 +300,17 @@ export class ChartTheme {
...
                 width: 0,
             },
         },
         log: {
             ...ChartTheme.getAxisDefaults(),
             base: 10,
             line: {
+                ...ChartTheme.getAxisDefaults().line,
+                // enabled: false, TODO: fix breaking tests when this is set, use width: 0 for now
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/themes/chartTheme.ts:313]: ?

**Review by alantreadway — COMMENTED** [review]
Code changes LGTM - are you planning on fixing the TODOs?

Can you make sure to capture breaking options changes in the JIRA under `Breaking Changes`?

**Pull request merged**